Formula/ffmpeg.rb

```diff
@@ -160,6 +160,12 @@ def install
       args << "--extra-ldflags=-L#{HOMEBREW_PREFIX}/include"
     end
 
+    if build.with? "jack"
+      ENV.prepend_path "PKG_CONFIG_PATH", Formula["jack"].opt_lib/"pkgconfig"
```

Comment:
If this doesn't work, try:

```suggestion
      ENV.prepend_path "PKG_CONFIG_PATH", Formula["jack"].opt_prefix/"lib64/pkgconfig"
```

Reply:
The original line seems to have worked for me under Linux:

```
➜ /home/linuxbrew/.linuxbrew/bin/ffmpeg -devices | grep jack
ffmpeg version 4.4 Copyright (c) 2000-2021 the FFmpeg developers
  built with gcc 11.2.0 (Homebrew GCC 11.2.0)
  configuration: --prefix=/home/linuxbrew/.linuxbrew/Cellar/ffmpeg/4.4-with-options_1 --enable-shared --cc=gcc-11 --host-cflags= --host-ldflags= --enable-gpl --enable-libaom --enable-libdav1d --enable-libmp3lame --enable-libopus --enable-libsnappy --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-libfontconfig --enable-libfreetype --enable-frei0r --enable-libass --enable-demuxer=dash --disable-htmlpages --enable-libjack --enable-indev=jack
  libavutil      56. 70.100 / 56. 70.100
  libavcodec     58.134.100 / 58.134.100
  libavformat    58. 76.100 / 58. 76.100
  libavdevice    58. 13.100 / 58. 13.100
  libavfilter     7.110.100 /  7.110.100
  libswscale      5.  9.100 /  5.  9.100
  libswresample   3.  9.100 /  3.  9.100
  libpostproc    55.  9.100 / 55.  9.100
 D  jack            JACK Audio Connection Kit
```

Reply:
Great; glad you're set then.

Reply:
Thanks for your help implementing this!